console/v1/0000_10_consoleplugin.crd.yaml

```diff
@@ -0,0 +1,288 @@
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  annotations:
+    api-approved.openshift.io: https://github.com/openshift/api/pull/1186
+    description: Extension for configuring openshift web console plugins.
+    displayName: ConsolePlugin
+    include.release.openshift.io/ibm-cloud-managed: "true"
+    include.release.openshift.io/self-managed-high-availability: "true"
+    include.release.openshift.io/single-node-developer: "true"
+  name: consoleplugins.console.openshift.io
+spec:
+  group: console.openshift.io
+  names:
+    kind: ConsolePlugin
+    listKind: ConsolePluginList
+    plural: consoleplugins
+    singular: consoleplugin
+  scope: Cluster
+  conversion:
+    strategy: Webhook
+    webhook:
+      conversionReviewVersions: ["v1", "v1alpha1"]
+      clientConfig:
+        service:
+          namespace: openshift-console-operator
+          name: conversion-webhook
+          path: /plugins-crd-convert
```

Comment:
Wondering does this need any authentication, TLS certs etc?

Reply:
good question.. but not sure how we should define caBundle here upfront?!... need to find out